account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -291,7 +332,9 @@ def _check_line_changed(self, line):
             or self.manual_account_id.id != line["account_id"][0]
             or self.manual_name != line["name"]
             or (
-                self.manual_partner_id and self.manual_partner_id.name_get()[0] or False
+                self.manual_partner_id
+                and self.manual_partner_id.name_get()[0]
+                or [False, False]
```

Comment:
`bool([False, False])` => `True`

Are you sure you want this?

Reply:
I think we compare this result to [False, False] yes, but I will check again

Reply:
If, from the reconciliation UI, for any bank statement line (which is not linked to any partner) you click on the counter part line, you can see that the onchange is called, and in the values of `reconcile_data_info` field, we have 
`[False, False]` in the `partner_id` key (in `data` list).
Then, if we compare here `False` with `[False, False]`, like it was before, it will be considered as a change where it is actually the same (there is no partner_id !).

That is why I did adapt here but I agree it is not nice and it could be an issue, since Odoo usually put `False` when there are not partner : https://github.com/OCA/account-reconcile/blob/16.0/account_reconcile_oca/models/account_reconcile_abstract.py#L69

But I don't know why the UI send `[False, False]` in the onchange, nor if it could be changed.
Any idea @etobella @yvaucher ?

Reply:
Well, it actually comes from https://github.com/OCA/account-reconcile/blob/17.0/account_reconcile_oca/models/account_bank_statement_line.py#L891

I'll do a commit to propose a change about it.

Reply:
Sorry for the noise here.
I'll make it short here. When there are not partner at all, in the data, the value  `[False, False]` is set here : 
https://github.com/OCA/account-reconcile/blob/16.0/account_reconcile_oca/models/account_bank_statement_line.py#L405
and here : 
https://github.com/OCA/account-reconcile/blob/16.0/account_reconcile_oca/models/account_bank_statement_line.py#L896

So when checking if the line has changed, and there is no partner at all, comparing the reconcile_data partner part (`[False, False]`)with `False` will always be considered different while no changed occured (no partner was set).

So I think we could leave this as is. (meaning comparing` [False, False]` with `line.get("partner_id")`